tests/acceptance/RobotLab/00_Smoke_Test.robot

```diff
@@ -15,6 +15,7 @@ Does RobotLab Load?
     [Arguments]    ${browser}
     [Documentation]    Does the Lab launcher show up with a Robot Framework entry?
     Set Tags    browser:${browser}
-    Open JupyterLab    browser=${browser}
+    Wait until keywords succeeds  3 x  1 s
+    ...  Open JupyterLab    browser=${browser}
```

Comment:
@bollwyvl This was required for Windows to fix issue where Selenium reported "No browser window open" for the first test suite only. Probably something happens too slow when Firefox cold starts.